MdePkg/Include/Library/PerformanceLib.h

```diff
@@ -40,7 +40,9 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define PERF_INMODULE_END_ID       0x41
 #define PERF_CROSSMODULE_START_ID  0x50
 #define PERF_CROSSMODULE_END_ID    0x51
-
+// MU_CHANGE
+#define PERF_VARIABLE_START_ID     0x60

```

Comment:
I would either do a MU_Change at the end of each line:

```
#define PERF_VARIABLE_START ID      0x60 # MU_CHANGE
```

Or do a MU_CHANGE begin, MU_CHANGE end